packages/clarity-js/src/core/scrub.ts

```diff
@@ -143,10 +147,12 @@ function regex(): void {
     // Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Unicode_Property_Escapes
     if (unicodeRegex && digitRegex === null) {
         try {
-            digitRegex = new RegExp("\\p{N}", "gu");
-            letterRegex = new RegExp("\\p{L}", "gu");
-            currencyRegex = new RegExp("\\p{Sc}", "gu");
-        } catch { unicodeRegex = false; }
+            digitRegex = /\p{N}/gu;
```

Comment:
when using a regex literal instead of a constructor the script will fail to load entirely if Unicode property escapes are not supported, rather than falling back gracefully - try-catch wont work